stylesheets/main.css

```
@@ -7,7 +7,7 @@
 
 	/* shuttle config */
 	--shuttle-logo: var(--shuttle-logo-jpl);
-	--rocket-fuel: 0;
+	--rocket-fuel: 11;
```

You need to set this variable to 1. 
```suggestion
	--rocket-fuel: 1;
```

---

Thanks, @gene-kranz